grails-app/domain/org/pih/warehouse/tablero/ColorNumber.groovy

```diff
@@ -0,0 +1,21 @@
+
+package org.pih.warehouse.tablero
+
+class ColorNumber implements Serializable {
+    
+    Long value;
+    String subtitle;
+
+    ColorNumber(Long value, String subtitle)
+    {
```

Comment:
This will probably be fixed once you apply the style guide, but I think these should be at the end of the previous line.